### DISCUSSION TITLE
お正月もランキングメール（先週いいねが多かった投稿ベスト20）があると嬉しい

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#595] [要望・アイデア・不具合]
毎年年末年始になると思うんですが、年始の第1週もランキングメールがあると嬉しいです。
もしメールを発行できなくても、第2週のメールで「先週のランキング」と「先々週のランキング」を2つ載せる、みたいになってるとありがたいです。

例：2022年12月〜2023年1月のランキングメール
<img width="976" alt="mail" src="https://github.com/increments/qiita-discussions/assets/1148320/01b2e604-e425-41b8-9e62-73019248cf1c">

## 理由
年末に書いた記事に結構たくさん「いいね」が付くことが過去に何度かあったのですが、年始の第1週にメールが送られないため、「これ、もしメールが送られていたら実はランキング入りしてたのでは？」と歯がゆい思いをしています。

100件のいいねが付いた記事があったとしても、公開したタイミングによってメールに載る・載らないが別れてしまうのは、記事の書き手にとってはちょっと悲しいです😢（ランキング入りすると、より多くの読者にリーチするので）

もちろん、Qiitaの中の人に「正月まで働け」とは言いませんが、上にも書いたように「第2週のメールで「先週のランキング」と「先々週のランキング」を2つ載せる」というような、何かしらの救済措置があるとありがたいな〜と思っています。

ご検討よろしくお願いします。

**Comment by tanaka-asami415** [comment]
@JunichiIto 
ご意見と詳細をありがとうございます！

早速社内で話し合い、1/5にランキング（計測期間:12/25～12/31）を記載して、メールを配信することにいたしました！✨
楽しみにお待ちいただけると幸いです！

**Reply by JunichiIto** [reply]
おお、素晴らしい！迅速な対応どうもありがとうございます！！ 🙏
